### ISSUE TITLE
Periodic check of links

### ISSUE THREAD

**Issue by enolfc** [EGI-Foundation/documentation#157]
# Short Description of the issue

Links should be periodically checked (specially those external to docs.egi.eu). See https://github.com/EGI-Foundation/fedcloud-catchall-operations/pull/24#discussion_r528575198

# Summary of proposed changes

Add periodic action on link checking

**Comment by brucellino** [comment]
Hi! I might be able to look at this as a quick exercise to get myself familiar. Are you thinking about a cron trigger here to run one of the actions included in the existing workflow, or an outside agent which monitors the pages for missing links and opens issues against them?

**Comment by enolfc** [comment]
There is already some regular checks of the links, but the result is an email that gets ignored in my inbox. See last execution https://github.com/EGI-Federation/documentation/actions/runs/7948069227. It would be good to open an issue indeed instead of just getting the notification about broke execution

**Comment by brucellino** [comment]
Hm. 14 minutes for a run. Sound like something you want to do asynch. I don't suppose we deploy this to a staging environment perhaps?

**Comment by enolfc** [comment]
15 minutes is checking every link on the repo. For PRs, only links within the changed files are checked, so it goes rather fast. Fixing the broken links will also reduce greatly the run (you just avoid hitting the timeouts for the broken links).

**Comment by enolfc** [comment]
yes and this is partially done (missing the opening issue if scheduled check fails)
See https://github.com/EGI-Federation/documentation/blob/main/.github/workflows/check-links.yml

**Comment by brucellino** [comment]
nice. So, todo is:

1. Open issue if cron fails
2. only check links in changed file which has changed in PR (Looks like that goes in `.github/workflows/build_pr_preview.yml` ?).. that sounds like a job for a pre-commit hook -- see #635
